### ISSUE TITLE
vue-for-range ruin scripts on IE11

### ISSUE THREAD

**Issue by chugunomi** [Krizzu/vue-for-range#2]
The following issue is caused by following symbol "`" in 
```js
console.warn(`Vue-for-range: provided argument for start - ${start} - is not a number.`)
```

After that IE says: `Invalid character` and stops running the js. So, the IE can't parse the grave accent, please use apostrophe `'` or quotes `"` symbol

**Comment by krizzu** [comment]
Hey @MikeChugunov 
Would you like to help and create a PR with the changes?

**Comment by chugunomi** [comment]
Done :)

**Comment by krizzu** [comment]
@MikeChugunov 
Hey, thanks! I left comments in PR

**Comment by krizzu** [comment]
Thanks! I've published new version, 1.0.2.
